config/web/rate-limit.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Yii\RateLimiter\CounterInterface;
+use Yiisoft\Yii\RateLimiter\Storage\StorageInterface;
+use \Yiisoft\Yii\RateLimiter\Counter;
+
+/** @var array $params */
+
+return [
+    StorageInterface::class => function () {
+        $cache = new \Yiisoft\Cache\File\FileCache(__DIR__ . '/../runtime/rate-limit/');
```

Comment:
How about to use `Yiisoft\Aliases\Aliases` with `@runtime` value?
[Here](https://github.com/yiisoft/demo/blob/master/config/params.php#L52) you can see all aliases

Reply:
[FileCache::__construct](https://github.com/yiisoft/cache-file/blob/master/src/FileCache.php#L105) does not support aliases

Reply:
And it should not.
[Here](https://github.com/yiisoft/demo/blob/de3b6fe1ae1d1a1429d5ed5f51aa467ec59a8714/config/common/translator.php#L15) is an example how to use `aliases`.
`$aliases->get('@runtime/rate-limiter')` === `/path/to/runtime/rate-limiter`

Reply:
Thanks